docs/DirectManipulation.md

```diff
@@ -83,11 +83,11 @@ performing continuous animations and responding to gestures, judiciously
 optimizing your components can improve your animations' fidelity.
 
 If you look at the implementation of `setNativeProps` in
-[NativeMethodsMixin.js](https://github.com/facebook/react-native/blob/master/Libraries/ReactIOS/NativeMethodsMixin.js)
+[NativeMethodsMixin.js](https://github.com/facebook/react/blob/master/src/renderers/native/NativeMethodsMixin.js)
```

Comment:
We can use direct link to line number here https://github.com/facebook/react/blob/master/src/renderers/native/NativeMethodsMixin.js#L142